web/.eslintrc.js

```diff
@@ -15,13 +15,38 @@
  * limitations under the License.
  */
 module.exports = {
-  extends: [require.resolve('@umijs/fabric/dist/eslint')],
+  extends: [
+    require.resolve('@umijs/fabric/dist/eslint'),
+    'plugin:import/errors',
+    'plugin:import/warnings',
+  ],
+  plugins: ['simple-import-sort'],
   globals: {
     ANT_DESIGN_PRO_ONLY_DO_NOT_USE_IN_YOUR_PRODUCTION: true,
     page: true,
     REACT_APP_ENV: true,
   },
   rules: {
     '@typescript-eslint/naming-convention': 'off',
+    'import/no-unresolved': [2, { ignore: ['^@/', '^umi/', '^@@/'] }],
+    'sort-imports': 'off',
+    'import/order': 'off',
```

Comment:
Why turn it off?

Reply:
Avoid conflict and according to simple-import-sort.

Reply:
Both parts of the modification look fine.